crates/lang/macro/src/lib.rs

```diff
@@ -665,6 +666,149 @@ pub fn trait_definition(attr: TokenStream, item: TokenStream) -> TokenStream {
     trait_def::analyze(attr.into(), item.into()).into()
 }
 
+/// Prepares the type to be fully compatible and usable with the storage.
+/// It implements all necessary traits and calculates the storage key for types.
+/// Packed types don't have a storage key, but non-packed types
```

Comment:
Can you link to the `Packed` trait docs?

Reply:
I can't because it is dev dependency